src/ideas/decorations/SlotMachine/idea.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "ac5855ff-d070-43a3-a29e-d18c66dbbac8",
+  "name": "Slot Machine",
+  "purpose": "Open a gambling hall.",
+  "schema": [
+    {
+      "name": "position",
+      "type": "position"
+    },
+    {
+      "name": "rotation",
+      "type": "rotation"
+    },
+    {
+      "name": "width",
+      "type": "number"
+    },
+    {
+      "name": "height",
+      "type": "number"
+    },
```

Comment:
this don't seem to be used?

Reply:
not sure how to do it but i would like to allow the slot machine to be rescaled